src/tests/functional/plugin/cpu/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

```diff
@@ -76,6 +76,10 @@ INSTANTIATE_TEST_SUITE_P(
         smoke_OVClassSetModelPriorityConfigTest, OVClassSetModelPriorityConfigTest,
         ::testing::Values("MULTI", "AUTO"));
 
+INSTANTIATE_TEST_SUITE_P(
+        smoke_OVClassSetAllowAutoBatchingConfigTest, OVClassSetAllowAutoBatchingConfigTest,
+        ::testing::Values("MULTI", "AUTO"));
```

Comment:
Please add other device like "CPU" and "GPU"

Reply:
pls use CommonTestUtils::DEVICE_MULTI, CommonTestUtils::DEVICE_CPU

Reply:
also pls add CommonTestUtils::DEVICE_BATCH

Reply:
I saw lots of device name using like "CPU" instead of CommonTestUtils::DEVICE_CPU in the test files of both folder ov_plugin and plugin. So should I update all of them or just update  the files only under the ov_plugin folder?

Reply:
you are only responsible to use right constanst in your code

Reply:
Got it